File: closeDebsta.py
import smtplib
from email.mime.text import MIMEText
from email.mime.multipart import MIMEMultipart
from datetime import datetime

email_address = "marshalgrossmanstaff@gmail.com"
app_password = ""

msg = MIMEMultipart()
msg["From"] = email_address
msg["To"] = email_address
msg["Subject"] = f"Daily Reminder - Close Debsta {datetime.now().strftime('%Y-%m-%d')}"

body = "For the love of God, close Debsta already"
msg.attach(MIMEText(body, "serif"))

with smtplib.SMTP("smtp.gmail.com", 587) as server:
    server.starttls()
    server.login(email_address, app_password)
    server.send_message(msg)

print("Email sent successfully!")

File: closeDebsta.js
function closeDebstaRemind() {
  const now = new Date();
  const day = now.getDay();
  const hour = now.getHours();
  const minute = now.getMinutes();

  if (day === 0 || day === 6) return;
  if (hour !== 16 || minute < 50) return;

  const props = PropertiesService.getScriptProperties();
  const today = now.toDateString();

  if (props.getProperty("lastSent") === today) return;

  sendDEBSTAEmail();
  props.setProperty("lastSent", today);
}

function sendDEBSTAEmail() {
  MailApp.sendEmail({
    to: "marshalgrossmanstaff@gmail.com",
    subject: "CLOSE DEBSTA | CLOSE DEBSTA",
    body: "For the love of God, close Debsta already!"
  });
}